pyproject.toml

```
@@ -228,20 +228,21 @@ ignore = [
 "docs/source/examples/notebooks/models/lithium-plating.ipynb" = ["F821"]
 
 [tool.pytest.ini_options]
-minversion = "6"
+minversion = "8"
+# Use pytest-xdist to run tests in parallel by default, exit with
+# error if not installed
 required_plugins = [
     "pytest-xdist",
     "pytest-mock",
 ]
 addopts = [
     "-nauto",
-    "-v",
-    "-ra",
+    "-vrA",
```

```suggestion
    "-vra",
```

I figured out the problem with the extra lines of logging output in the tests for the example scripts – this slipped right under our noses. This should be `-a`, not `-A` – the latter prints out the outputs from the tests that passed, and not just their names. Using `-a` should reduce the length of the logs by a significant amount.